packages/sdk/src/Base.ts

```diff
@@ -429,12 +429,18 @@ class Base {
     return txOptions
   }
 
-  public getBonderAddress (token: TToken): string {
+  protected _getBonderAddress (token: TToken, sourceChain: TChain, destinationChain?: TChain): string {
     token = this.toTokenModel(token)
-    return this.bonders?.[this.network]?.[token.canonicalSymbol]?.[0]
+    sourceChain = this.toChainModel(sourceChain)
+    destinationChain = this.toChainModel(destinationChain)
+    const sourceBonders = this.bonders?.[this.network]?.[token.canonicalSymbol]?.[sourceChain.slug]
+    if (destinationChain) {
```

Comment:
With the introduction of bonder routes, I _think_ we now need to always have the context of source and destination for stuff like this. Thinking out loud, this is probably the biggest mental model update.

Reply:
Resolved by https://github.com/hop-protocol/hop/commit/98c910a15e4b6e3537df362b44283f62f2164323